static/templates/actors/character/limited.hbs

```diff
@@ -46,7 +46,7 @@
             <div class="tab actions actions-pane" data-group="primary" data-tab="effects">
                 <ol class="actions-list directory-list strikes-list">
                     <li class="item action-header stroke-header">
-                        {{> systems/pf2e/templates/actors/partials/images/header_stroke.hbs}}
+                        {{> "systems/pf2e/templates/actors/partials/images/header_stroke.hbs" }}
```

Comment:
```suggestion
                        {{> "systems/pf2e/templates/actors/partials/images/header_stroke.hbs"}}
```
:o)

Reply:
I see you don't like symmetry

Reply:
I love it! the > is the first character inside the curly braces, and it's touching.

Reply:
done but I didn't test it